Trying locally `mvn verify -Dnative -Dtest-kafka` and I had one run which passed fully and another run which passed on test but Maven is reporting failure. Using GraalVM CE 19.3.1 jdk11

Similar situation happened on CI which reports `[INFO] Tests run: 9, Failures: 0, Errors: 0, Skipped: 0`
But Maven says `[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.1:test (default-test) on project quarkus-integration-test-kafka: There are test failures.`

Some details from my local run
```
...
[INFO] Tests run: 9, Failures: 0, Errors: 0, Skipped: 0
...
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO]
[INFO] --- maven-failsafe-plugin:2.22.1:verify (default) @ quarkus-integration-test-kafka ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  05:12 min
[INFO] Finished at: 2020-04-29T11:22:27+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:2.22.1:verify (default) on project quarkus-integration-test-kafka: There are test failures.
[ERROR]
[ERROR] Please refer to /Users/rsvoboda/git/quarkus/integration-tests/kafka/target/failsafe-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] org.apache.maven.surefire.booter.SurefireBooterForkException: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /Users/rsvoboda/git/quarkus/integration-tests/kafka && /Users/rsvoboda/.sdkman/candidates/java/19.3.1.r11-grl/bin/java -jar /Users/rsvoboda/git/quarkus/integration-tests/kafka/target/surefire/surefirebooter9517779100608106166.jar /Users/rsvoboda/git/quarkus/integration-tests/kafka/target/surefire 2020-04-29T11-21-50_851-jvmRun1 surefire15503831576610706916tmp surefire_012153274959446584857tmp
[ERROR] Error occurred in starting fork, check output in log
[ERROR] Process Exit Code: 1

```

--- a/.github/workflows/ci-actions.yml
+++ b/.github/workflows/ci-actions.yml
@@ -34,8 +34,8 @@ on:
 env:
   # Workaround testsuite locale issue
   LANG: en_US.UTF-8
-  NATIVE_TEST_MAVEN_OPTS: "-B --settings .github/mvn-settings.xml --fail-at-end -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java11 -Dtest-postgresql -Dtest-elasticsearch -Dtest-keycloak -Dtest-dynamodb -Dtest-mysql -Dtest-mariadb -Dmariadb.url='jdbc:mariadb://localhost:3308/hibernate_orm_test'  -Dtest-mssql -Dtest-vault -Dtest-neo4j -Dnative-image.xmx=5g -Dnative -Dformat.skip install"
-  JVM_TEST_MAVEN_OPTS: "-e -B --settings .github/mvn-settings.xml -Dtest-postgresql -Dtest-elasticsearch -Dtest-mysql -Dtest-mariadb -Dmariadb.url='jdbc:mariadb://localhost:3308/hibernate_orm_test'  -Dtest-mssql -Dtest-dynamodb -Dtest-vault -Dtest-neo4j -Dtest-keycloak -Dformat.skip"
+  NATIVE_TEST_MAVEN_OPTS: "-B --settings .github/mvn-settings.xml --fail-at-end -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java11 -Dtest-postgresql -Dtest-elasticsearch -Dtest-keycloak -Dtest-dynamodb -Dtest-mysql -Dtest-mariadb -Dmariadb.url='jdbc:mariadb://localhost:3308/hibernate_orm_test'  -Dtest-mssql -Dtest-vault -Dtest-neo4j -Dtest-kafka -Dnative-image.xmx=5g -Dnative -Dformat.skip install"
+  JVM_TEST_MAVEN_OPTS: "-e -B --settings .github/mvn-settings.xml -Dtest-postgresql -Dtest-elasticsearch -Dtest-mysql -Dtest-mariadb -Dmariadb.url='jdbc:mariadb://localhost:3308/hibernate_orm_test'  -Dtest-mssql -Dtest-dynamodb -Dtest-vault -Dtest-neo4j -Dtest-kafka -Dtest-keycloak -Dformat.skip"
   DB_USER: hibernate_orm_test
   DB_PASSWORD: hibernate_orm_test
   DB_NAME: hibernate_orm_test

--- a/.github/workflows/native-cron-build.yml
+++ b/.github/workflows/native-cron-build.yml
@@ -54,7 +54,7 @@ jobs:
         run: mvn -B install -DskipTests -DskipITs -Dformat.skip
 
       - name: Run integration tests in native
-        run: mvn -B --settings .github/mvn-settings.xml verify -f integration-tests/pom.xml --fail-at-end -Dno-format -Ddocker -Dnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java${{ matrix.java }} -Dtest-postgresql -Dtest-elasticsearch -Dtest-mysql -Dtest-dynamodb -Dtest-vault -Dtest-neo4j -Dtest-keycloak -Dtest-mssql -Dtest-mariadb -Dmariadb.url="jdbc:mariadb://localhost:3308/hibernate_orm_test"
+        run: mvn -B --settings .github/mvn-settings.xml verify -f integration-tests/pom.xml --fail-at-end -Dno-format -Ddocker -Dnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java${{ matrix.java }} -Dtest-postgresql -Dtest-elasticsearch -Dtest-mysql -Dtest-dynamodb -Dtest-vault -Dtest-neo4j -Dtest-keycloak -Dtest-kafka -Dtest-mssql -Dtest-mariadb -Dmariadb.url="jdbc:mariadb://localhost:3308/hibernate_orm_test"
 
       - name: Report
         if: always()

--- a/integration-tests/kafka/pom.xml
+++ b/integration-tests/kafka/pom.xml
@@ -184,16 +184,22 @@
 
     <profiles>
         <profile>
-            <id>docker</id>
+            <id>test-kafka</id>
             <activation>
                 <property>
-                    <name>docker</name>
+                    <name>test-kafka</name>
                 </property>
             </activation>
             <build>
                 <plugins>
                     <plugin>
                         <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <skip>false</skip>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <artifactId>maven-failsafe-plugin</artifactId>
                         <configuration>
                             <skip>false</skip>
                         </configuration>
@@ -221,7 +227,6 @@
                                     <goal>verify</goal>
                                 </goals>
                                 <configuration>
-                                    <skip>false</skip>
                                     <systemProperties>
                                         <native.image.path>${project.build.directory}/${project.build.finalName}-runner
                                         </native.image.path>

--- a/integration-tests/kafka/src/test/java/io/quarkus/it/kafka/KafkaSASLTestResource.java
+++ b/integration-tests/kafka/src/test/java/io/quarkus/it/kafka/KafkaSASLTestResource.java
@@ -23,7 +23,7 @@ public class KafkaSASLTestResource implements QuarkusTestResourceLifecycleManage
     @Override
     public Map<String, String> start() {
         try {
-            File directory = Testing.Files.createTestingDirectory("kafka-data", true);
+            File directory = Testing.Files.createTestingDirectory("sasl-kafka-data", true);
             File sslDir = sslDir(directory, true);
 
             Path ksPath = new File(sslDir, "kafka-keystore.p12").toPath();
@@ -82,7 +82,7 @@ public class KafkaSASLTestResource implements QuarkusTestResourceLifecycleManage
 
     public static File sslDir(File directory, boolean removeExistingContent) throws IOException {
         if (directory == null) {
-            directory = Testing.Files.createTestingDirectory("kafka-data", removeExistingContent);
+            directory = Testing.Files.createTestingDirectory("sasl-kafka-data", removeExistingContent);
         }
 
         File targetDir = directory.getParentFile().getParentFile();

